Makefile

```diff
@@ -35,3 +35,7 @@ clean-data:
 .PHONY: describe-data
 describe-data:
 	python -m ${BASEDIR} --describe-data
+
+.PHONY: logistic-regression
```

Comment:
Add this command to the `run` one above it, por favor.

Reply:
Could you also add it to the list of commands in the `README.md` file?

Reply:
sure